oshmem/include/pshmem.h

```diff
@@ -245,6 +245,10 @@ OSHMEM_DECLSPEC void pshmem_collect32(void *target, const void *source, size_t n
 OSHMEM_DECLSPEC void pshmem_collect64(void *target, const void *source, size_t nlong, int PE_start, int logPE_stride, int PE_size, long *pSync);
 OSHMEM_DECLSPEC void pshmem_fcollect32(void *target, const void *source, size_t nlong, int PE_start, int logPE_stride, int PE_size, long *pSync);
 OSHMEM_DECLSPEC void pshmem_fcollect64(void *target, const void *source, size_t nlong, int PE_start, int logPE_stride, int PE_size, long *pSync);
+OSHMEM_DECLSPEC void pshmem_alltoall32(void *target, const void *source, size_t nelems, int PE_start, int logPE_stride, int PE_size, long *pSync);
```

Comment:
i did not see that in master, so i committed it in open-mpi/ompi@cb84f582b282232a412816686986fdf9d05b9869